contrib/static-opus/Makefile

```diff
@@ -37,25 +39,50 @@ LDFLAGS:= $(filter-out -funsafe-math-optimizations,$(LDFLAGS))
 #  Common commands and arguments
 #  -----------------------------
 THREADS = $(shell nproc || echo 4)
-CURL = curl -s -L
+CURL = curl --location-trusted --insecure --progress-bar
+WGET = wget --no-clobber --no-check-certificate --no-verbose --progress=bar
```

Comment:
Suggestion: maybe we should put user-overridable variables in here, like:

    CURL = curl $(CURL_FLAGS)

Then set defaults to safe behaviour, and provide the documentation (in comment) on how to trigger unsafe behaviour?

Reply:
Oh, that's a great idea. I'll add the certificate exclusion directly in the error message, and this knocks the need to mention the security issue having now moved it into the users' court.
(squashed and updated the PR text)